raisinbread/RB_files/RB_Config.sql

```diff
@@ -92,9 +92,12 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (95,49,'1000');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (96,100,'environment');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (97,70,'/data-raisinbread/');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (98,93,'V1');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (99,101,'');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (102,19,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (99,65,'true');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (100,19,'false');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (101,65,'true');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (102,101,'Y-m-d H:i:s');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (103,102,'/data/document_repository_uploads/');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (104,103,'/data/data_release_uploads/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (105,104,'Y-m-d H:i:s');
```

Comment:
you have this config duplicated on line 98

Reply:
Thanks, I'll redo the RB export